pom.xml

```diff
@@ -113,6 +113,12 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
+				<exclusions>
```

Comment:
This avoids some strange method not found exception during tests (`JsonObject.toMap()`)

Reply:
BTW: That confuses me a little. I believe to remember that I had a very similar issue w.r.t. the same method name - and that was some very obscure, but important version clash (could have caused other severe issues as well). Will try to have a look at this, if time permits.

Reply:
I found it. Please see 45a2cfd632e9ed7140d106a674b9e07696f3c773 for the trick...